lib/connection/result/row_stream.js

```diff
@@ -428,17 +437,27 @@ function buildMapColumnExtractFnNames(columns, fetchAsString)
  * @param {Object} row
  * @param {Object[]} columns
  * @param {Object} [mapColumnIdToExtractFnName]
+ * @param {String?} rowMode - "array" or "object", default is "object"
```

Comment:
in other comment type is described as `the rowMode ('array' or 'object') string`